plugins/generator-1.20.4/neoforge-1.20.4/templates/elementinits/items.java.ftl

```diff
@@ -91,6 +93,23 @@ public class ${JavaModName}Items {
 		</#if>
 	</#list>
 
+	<#if itemsWithInventory?size != 0>

```

Comment:
No need to wrap list with size check if the only thing inside if statement is the list itself

Reply:
Inside there is list of constants **AND** method with list of registration statements.

Reply:
There also should be registry constant declaration with the previous note taken into account.

Reply:
I see, what got me confused is you use two different indentation approaches for FTL tags in the same location:

![image](https://github.com/MCreator/MCreator/assets/16374228/56d8e286-8090-44c4-979e-1f045f073861)

Please fix this part in this case